src/runtime/Makefile

```diff
@@ -446,11 +446,11 @@ ifneq (,$(QEMUCMD))
     DEFAULTVFIOPORT_NV = root-port
     DEFAULTPCIEROOTPORT_NV = 8
 
-    KERNELPARAMS_NV =  "agent.hotplug_timeout=20"
-    KERNELPARAMS_NV += $(KERNELPARAMS)
-
-    KERNELTDXPARAMS_NV = "authorize_allow_devs=pci:ALL"
-    KERNELTDXPARAMS_NV += $(KERNELTDXPARAMS)
+    KERNELPARAMS_NV =  "agent.hotplug_timeout=20 cgroup_no_v1=all"
```

Comment:
Do you explicitly not want `systemd.unified_cgroup_hierarchy=1` now?

Reply:
Do not need it. and may confuse people in thinking that the GPU initrd/image is using systemd, which we are not.